### PR TITLE
Update README and add design document

### PR DESCRIPTION
**Pull request by naturetime10**

### CODE REVIEW

File: design/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+![Day of](Day-of.jpg)
+
+![Map](Map.jpg)
+
+## Editing
+
+You need a Mac with [Sketch](https://www.sketchapp.com) installed to open the document.
+
+## Contributing
```
Comment (binamkayastha) [inline design/README.md:22]: Should this be in the top level readme?

File: design/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or Facebook group chat with the owners of this repository before making a change.
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a build.
+
+2. Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations and container parameters.
+
+3. Increase the version numbers in any examples files and the README.md to the new version that this Pull Request would represent. The versioning scheme we use is SemVer.
+
+4. You may merge the Pull Request in once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
```
Comment (binamkayastha) [inline design/README.md:31]: Lol we don't have enough developers to have two reviewers. You can probably just remove the whole number

File: design/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Authors
+
+- **Yang Liu** - *Initial work* - [byliuyang](https://github.com/byliuyang)
+
+## License
+This design document is licensed under the Creative Commons 4.0 - see the [LICENSE.md](LICENSE.md) file for details
```
Comment (binamkayastha) [inline design/README.md]: Please change to GPL 2

**Review by binamkayastha — CHANGES_REQUESTED** [review]
Nice work! Thanks for the great design! See comments below.

File: design/README.md (hunk 2 above)
Comment (binamkayastha) [reply design/README.md:31]: Talked this over with Harry, this is specifically for design documents, so both Harry and I can review them.

File: design/README.md (hunk 1 above)
Comment (binamkayastha) [reply design/README.md:22]: Talked this over with Harry, this secion is specifically for the design documents.

**Comment by binamkayastha** [comment]
Out of curiosity, did you ever implement the design document?

**Comment by naturetime10** [comment]
Yes, I did.

**Pull request closed without merging**